Fabric.Identity.API/scripts/Install-Identity.ps1

```diff
@@ -150,6 +136,67 @@ if ($fabricInstallerSecret){
     Write-DosMessage -Level "Information" -Message "Fabric.Installer clientSecret has been created."
 }
 
+
+# install IdPSS TODO move out to IdPSS ps1 script #######################################################################
+# Get Idpss app pool user 
+# Create log directory with read/write permissions for app pool user
+$idpssSettingsScope = "idpss"
+$idpssServiceUrl = Get-ApplicationEndpoint -appName $idpssConfigStore.appName -applicationEndpoint $idpssConfigStore.applicationEndPoint -installConfigPath $installConfigPath -scope $idpssSettingsScope -quiet $quiet
+    
+$idpssWebDeployParameters = Get-WebDeployParameters -serviceConfig $idpssConfigStore -commonConfig $commonConfigStore
+$idpssStandalonePath = ".\Fabric.IdentityProviderSearchService.zip"
+$idpssInstallerPath = "..\WebDeployPackages\Fabric.IdentityProviderSearchService.zip"
+$idpssInstallPackagePath = Get-WebDeployPackagePath -standalonePath $idpssStandalonePath -installerPath $idpssInstallerPath
+
+if($null -eq $credential)
```

Comment:
if credential isn't null, it's still trying to use the $idpssIISUser set inside the if, so it will be null down below in the publish portion right?

Reply:
good catch. updating...